hermes-console-vue/src/components/ace-editor/AceEditor.vue

```diff
@@ -0,0 +1,58 @@
+<script setup lang="ts">
+  import '@/components/ace-editor/ace-config';
+  import { defineProps } from 'vue';
+  import { useTheme } from 'vuetify';
+  import { VAceEditor } from 'vue3-ace-editor';
+  import { useGlobalI18n } from '@/i18n';
+  import { useNotificationsStore } from '@/store/app-notifications/useAppNotifications';
+
+  const theme = useTheme();
+  const notificationStore = useNotificationsStore();
+
+  const { t } = useGlobalI18n();
+  const props = defineProps<{
+    modelValue: string;
+    label?: string;
+    placeholder?: string;
+  }>();
+  const emit = defineEmits(['update:modelValue']);
+  const beautify = () => {
+    try {
+      const obj_message = JSON.parse(props.modelValue || '');
+      emit('update:modelValue', JSON.stringify(obj_message, null, 4));
+    } catch (e) {
+      notificationStore.dispatchNotification({
+        title: t('notifications.form.validationError'),
```

Comment:
title is not necessary, also it says about vaildationError, so it can be removed